env_files/config.env

```diff
@@ -5,13 +5,13 @@
 # over time to construct other
 # values required by this repository
 #####################################################
-CONFLUENT=6.2.0-0
-CONFLUENT_DOCKER_TAG=6.2.x-latest
-CONFLUENT_SHORT=6.2
+CONFLUENT=6.1.0-0
+CONFLUENT_DOCKER_TAG=6.1.x-latest
+CONFLUENT_SHORT=6.1
 CONFLUENT_PREVIOUS=""
 CONFLUENT_RELEASE_TAG_OR_BRANCH=master
 CONFLUENT_MAJOR=6
-CONFLUENT_MINOR=2
+CONFLUENT_MINOR=1
```

Comment:
Sorry if i'm jumping the gun, I know this marked Draft, but i don't think any change should modify this file

Reply:
@rspurgeon i wasn't planning on merging this change, it was just for testing on this branch.